designsafe/sitemaps.py

```diff
@@ -193,7 +194,7 @@ def items(self):
                         'project' : proj['path'],
                         'system' : proj['system']
                     }
-                    projPath.append('{root}public/{system}/{project}'.format(**subpath))
+                    projPath.append('{root}public/{system}{project}'.format(**subpath))
                 else:
```

Comment:
I was a little confused with this part. If `project` in this case is the project id `project: "PRJ-XXXX"` why are we removing the `/` between `PRJ-XXXX` and `designsafe.storage.published`? I'm only asking because in the href of `base.html` we still have the slash after the storage system and before the projectId like so:
`https://www.designsafe-ci.org/data/browser/public/designsafe.storage.published/{{projectId}}`

Reply:
This is for NEES projects, which includes a leading slash.  One of Four Kitchen's asks is to have only a single slash for these sitemap URLs.  This is so the canonical URLs match the sitemap URLs, which would match the project's URL.  So, to make it a single slash in the sitemap, I had to get rid of the slash here.